starboard/contrib/RDK/README.md

```diff
@@ -0,0 +1,17 @@
+### RDK Cobalt Open Source reference port
+
+RDK Starboard reference port for the RDK-V profile is maintained by the RDK team.
+It is currently targeted to build on Amlogic AH212 reference device.
+
```

Comment:
Better to include the details of the Manifest file, this will help to speed up the cobalt build process on Yocto based RDK system.
Here is the recipe details :
https://code.rdkcentral.com/r/plugins/gitiles/rdk/components/generic/rdk-oe/meta-rdk-cobalt

Reply:
I'm not sure i entirely understood as to what should be said here in regards to manifest file. I can put in a `meta-rdk-cobalt` build manifest link, sure, but i'm not sure how would it help the reader ? Is there a longer sentence to include as to what this does or how to use it ?

For context, "Starboard" and Starboard code should be a familiar context for anyone reading this, as it lives in `starboard/config`, but i'm not sure open-embedded meta layer would be. @alexanderahn for comment as well.

Reply:
A sentence on how it would help with the build process on the RDK systems would help me better understand the purpose as well. Adding a link for now would be the minimum.

Reply:
ping for @shibupthomas for clarification.

Reply:
Actually i'll go ahead an merge this as is for now, and follow up in a new PR